README.md

```diff
@@ -1,5 +1,18 @@
 WebGL 2 Samples Pack
 =====================
 
-Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://www.linkedin.com/in/shuai-shao-3718818b) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania. 
+GH-Page: http://webglsamples.org/WebGL2Samples/
 
+Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://shrekshao.github.io) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania. 
+
+## Samples list
+
+|              | Chrome 47 Win | Firefox 42 Win|
+|--------------|---------------|---------------|
+|draw instanced|   ✓    |  crash        |
+
+
+## Credits
+* [three.js example page framework](https://github.com/mrdoob/three.js)
```

Comment:
Are you using code from any of these three projects?  If so, please including their license in a third-party section of a LICENSE.md file.  For example: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md